manifests/facts.pp

```diff
@@ -32,6 +32,9 @@
   if $facts['puppetversion'] =~ /Puppet Enterprise/ {
     $ruby_bin = '/opt/puppet/bin/ruby'
     $dir = 'puppetlabs/'
```

Comment:
I think this part is redundant now, assuming the other comment is correct.

Reply:
I don't have Puppet Enterprise to be able to test it to know for sure.